pkg/operator/apiserver/controller/workload/workload_test.go

```diff
@@ -105,8 +107,8 @@ func TestUpdateOperatorStatus(t *testing.T) {
 					{
 						Type:    fmt.Sprintf("%sWorkloadDegraded", defaultControllerName),
 						Status:  operatorv1.ConditionTrue,
-						Message: "nasty error\n",
-						Reason:  "SyncError",
+						Reason:  "NoDeployment",
```

Comment:
We shouldn't overwrite `SyncError` if occurred, these errors are important. Does that make sense?

Reply:
Good point. I added a commit to change that, PTAL.